scintillometry/fourier/base.py

```diff
@@ -151,13 +151,20 @@ def freq(self):
         Returns
         -------
         freqs : `~numpy.ndarray` or array of `~astropy.units.Quantity`
-            Sample frequencies.
+            Sample frequencies, with shape (len(f), 1, ..., 1).  The trailing
+            dimensions of length unity are to facilitate broadcasting when
+            operating on ``freqs``.
         """
         sample_rate = 1. if self.sample_rate is None else self.sample_rate
         a_length = self._time_shape[self.axis]
         if self._time_dtype.kind == 'f':
-            return np.fft.rfftfreq(a_length, d=(1. / sample_rate))
-        return np.fft.fftfreq(a_length, d=(1. / sample_rate))
+            freqs = np.fft.rfftfreq(a_length, d=(1. / sample_rate))
+        else:
+            freqs = np.fft.fftfreq(a_length, d=(1. / sample_rate))
+        # Add trailing dimensions.
+        freqs_shape = (freqs.shape +
```

Comment:
In this case, it may be quite reasonable to do the change in-place, i.e.,
```
freqs.shape = (freqs.shape + ....
```

Reply:
You can do that!?  Changing...